lib/optimizely.rb

```diff
@@ -282,6 +282,28 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
       false
     end
 
+    def get_enabled_features(user_id, attributes = nil)
+      # Returns the list of features that are enabled for the user.
```

Comment:
Get keys of all feature flags which are enabled for the user

Reply:
Done